pkg/deb/postinstall

```diff
@@ -2,3 +2,9 @@
 mkdir -p /var/log/serviced
 chmod 1777 /var/log/serviced
 
+for BINARY in "serviced" "serviced-controller" "serviced-storage"; do
+    chgrp serviced /opt/serviced/bin/${BINARY}
+done
+
+chgrp serviced /etc/default/serviced
+chgrp serviced /opt/serviced
```

Comment:
There's a loop above that changes three specific files to have `root:serviced` ownership, but it seems inconsistent to have different ownership in the `/opt/serviced` directory.

Changing the above line to `chgrp -R serviced /opt/serviced` will switch everything over to the `serviced` group and eliminate the need for the earlier loop.